static/js/components/CourseList.js

```diff
@@ -29,37 +43,75 @@ class CourseList extends React.Component {
           coursesPassed++;
         }
 
-        return <ul
+        let courseRuns = [];
+        if (expander[course.id]) {
+          courseRuns = course.runs.map(run => {
+            return <Grid style={{height: DASHBOARD_RUN_HEIGHT}} key={run.id}>
+              <Cell
+                col={6}
+                className="course-run-title"
+              >
+                {run.title}
+              </Cell>
+              <Cell col={4} className="course-run-status">
+                {makeRunStatusDisplay(run)}
+              </Cell>
+            </Grid>;
+          });
+        }
+
+        let expanderClass = "glyphicon glyphicon-chevron-down";
+        if (expander[course.id]) {
+          expanderClass = "glyphicon glyphicon-chevron-up";
+        }
```

Comment:
suggestion: change this to

```js
let setExpanderIcon = () => (
  expander[course.id] ? "glyphicon-chevron-up" : "glyphicon-chevron-down"
);
```

then when we need this class later just do

```js
className={`glyphicon ${setExpanderIcon}`}
```

or alternatively I guess we could just do

```js
let expanderClass = `glyphicon glyphicon-chevron-${expander[course.id] ? "up" : "down"}`;
```

that could also just be stuck directly in `className`. hmmm....